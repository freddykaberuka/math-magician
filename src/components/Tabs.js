import React from 'react';
import { Link } from 'react-router-dom';

function Tabs() {
  return (
    <div className="header">
      <h1>Math Magicians</h1>
      <nav style={{ padding: '40px', textDecoration: 'none', fontSize: '32px' }}>
        <Link to="/">Home</Link>
        {' '}
        |
        <Link to="/calculator">Calculator</Link>
        {' '}
        |
        <Link to="/quote">Quote</Link>
      </nav>
    </div>
  );
}

export default Tabs;
