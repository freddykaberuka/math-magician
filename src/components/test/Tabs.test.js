import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import Tabs from '../Tabs';

describe('Test for rendering the navigation', () => {
  it('It should render the navigation header bar', () => {
    const tabs = renderer
      .create(<Router><Tabs /></Router>)
      .toJSON();
    expect(tabs).toMatchSnapshot();
  });
});
