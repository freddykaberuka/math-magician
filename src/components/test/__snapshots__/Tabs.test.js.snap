// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for rendering the navigation It should render the navigation header bar 1`] = `
<div
  className="header"
>
  <h1>
    Math Magicians
  </h1>
  <nav
    style={
      Object {
        "fontSize": "32px",
        "padding": "40px",
        "textDecoration": "none",
      }
    }
  >
    <a
      href="#/"
      onClick={[Function]}
    >
      Home
    </a>
     
    |
    <a
      href="#/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
     
    |
    <a
      href="#/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</div>
`;
